### ISSUE TITLE
Are we actually using font awesome? 

### ISSUE THREAD

**Issue by phillipadsmith** [TheTyee/design-article.thetyee.ca#189]
@alexgreen Do you know if we're actually referencing anything in Font Awesome?

**Comment by phillipadsmith** [comment]
@MrBryan said yes, in the header, for the share icons.

@alexgreen If this is the only use, I would propose we move those icons to .png files and get rid of the loading of the Font Awesome fonts in the interest of performance.

**Comment by alexgreen** [comment]
Yes if it improves performance lets do that. I think I only used it because it was bundled with bootstrap.

**Comment by phillipadsmith** [comment]
> Yes if it improves performance lets do that. I think I only used it because it was bundled with bootstrap.

Great. Would you be open to starting a feature branch, something like `remove-font-awesome` and proposing those changes there? No rush, as we can always do that launch.

**Comment by alexgreen** [comment]
ok
